spec/index.bs

```diff
@@ -1761,6 +1788,80 @@ enum GPUCompareFunction {
 };
 </script>
 
+<dl dfn-type="enum-value" dfn-for=GPUCompareFunction>
+    : <dfn>"never"</dfn>
+    ::
+        Comparison tests never pass.
+
+    : <dfn>"less"</dfn>
+    ::
+        A provided value passes the comparison test if it is less than the stored value.
+
+    : <dfn>"equal"</dfn>
+    ::
+        A provided value passes the comparison test if it is equal to the stored value.
+
+    : <dfn>"less-equal"</dfn>
+    ::
+        A provided value passes the comparison test if it is less than or equal to the stored value.
+
+    : <dfn>"greater"</dfn>
+    ::
+        A provided value passes the comparison test if it is greater than the stored value.
+
+    : <dfn>"not-equal"</dfn>
+    ::
+        A provided value passes the comparison test if it is not equal to the stored value.
+
+    : <dfn>"greater-equal"</dfn>
+    ::
+        A provided value passes the comparison test if it is greater than or equal to the stored value.
+
+    : <dfn>"always"</dfn>
+    ::
+        Comparison tests always pass.
+</dl>
+
+<div algorithm>
+    <dfn abstract-op>validating GPUSamplerDescriptor</dfn>(device, descriptor)
+    **Arguments:**
+        - {{GPUDevice}} |device|
+        - {{GPUSamplerDescriptor}} |descriptor|
+    
+    **Returns:** boolean
+
+    Return true if and only if all of the following conditions apply:
+        - |device| is valid.
+        - |descriptor|.{{GPUSamplerDescriptor/lodMinClamp}} is greater than or equal to 0.
+        - |descriptor|.{{GPUSamplerDescriptor/lodMaxClamp}} is greater than or equal to
```

Comment:
This should also disallow `NaNs`.

Reply:
noob q, do IDL bindings for float not prevent this?

Reply:
> Infinity, -Infinity and NaN must not be used as the value of a float or double.

> The double type is a floating point numeric type that corresponds to the set of finite double-precision 64 bit IEEE 754 floating point numbers. [IEEE-754]

> The unrestricted double type is a floating point numeric type that corresponds to the set of all possible double-precision 64 bit IEEE 754 floating point numbers, finite, non-finite, and special "not a number" values (NaNs). [IEEE-754]